clustered/server/src/main/java/org/ehcache/clustered/server/store/ClusterTierActiveEntity.java

```diff
@@ -588,12 +600,11 @@ public void synchronizeKeyToPassive(PassiveSynchronizationChannel<EhcacheEntityM
         mappingsToSend.set(new HashMap<>());
         size.set(0);
       }
-    }
-
-    // Sync message tracker as well, to prevent duplicates received on the passive
-    // We do it once on the last concurrency key
-    if (concurrencyKey == lastConcurrencyKey) {
-      syncChannel.synchronizeToPassive(new EhcacheMessageTrackerMessage(messageHandler));
+      Map<Long, Map<Long, EhcacheEntityResponse>> clientSourceIdTrackingMap = messageHandler.getTrackedClients()
```

Comment:
Shouldn't the segment index arguments in this line and the next line be the same as the `segmentId` field rather than just `concurrencyKey - 1`?

Reply:
No because we use the first segment aka `0` for state repository info and `1` to `16` (currently) for store segmenting.

Reply:
Well ignore this. Just realised that DEFAULT_KEY is 1. So this is still fine.